docs/README.md

```diff
@@ -83,26 +86,39 @@ This repository provides blocks that can help you create a Dropdown, Select or a
 | `'visible'` | Displays a backdrop overlay which blocks the click on other items of the page, if you click on the overlay it will close the popover |
 | `'none'` | Doesn't display any backdrop overlay and if you click any item in the page it will close the popover and click the element |
 
+### Offsets
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `skidding` | `number` | Displaces the `popover-layout` along the `popover-trigger`. |
+| `distance` | `number` | Displaces the `popover-layout` away from, or toward, the `popover-trigger` in the direction of its placement. A positive number displaces it further away, while a negative number lets it overlap the `popover-trigger`. |
+
+
 ### PopoverTrigger
 
 | Prop name | Type | Description | Default value |
 | --- | --- | --- | --- |
-| trigger | `TriggerMode` | What kind of user action must happen to open the popover | `'click'` |
+| trigger | `TriggerMode` | What kind of user action must happen to open the `popover-layout` | `'click'` |
```

Comment:
```suggestion
| trigger | `TriggerMode` | What kind of user action must happen to open the `popover-layout`.<br/> - `click`<br/>- `hover` | `'click'` |
```

Reply:
I think we could talk with @guerreirobeatriz about how to do this and change the structure of the documentation in general